web-client/src/presenter/computeds/contactsHelper.js

```diff
@@ -15,319 +15,18 @@ export const contactsHelper = (get, applicationContext) => {
   const { PARTY_TYPES, USER_ROLES } = applicationContext.getConstants();
 
   let contactPrimary, contactSecondary;
-  let showEmail = true;
+  let showEmail = user.role !== USER_ROLES.petitioner;
```

Comment:
looks like this is now a constant, and can be immediately put to use at line 20. :)